internal/model/event.go

```diff
@@ -0,0 +1,8 @@
+package model
+
+import "gorm.io/gorm"
+
+type Event struct {
```

Comment:
I would like to see something like annotate in ruby, because idk what fields are created by default with gorm.Model

Reply:
gorm.Model it inherited struct what have 
```
// gorm.Model definition
type Model struct {
  ID        uint           `gorm:"primaryKey"`
  CreatedAt time.Time
  UpdatedAt time.Time
  DeletedAt gorm.DeletedAt `gorm:"index"`
}
--



```